generator/template/ready-to-use-section-types/components/UnifiedMenu/ItemsGrid.vue

```diff
@@ -29,9 +29,32 @@
             <!-- Item Name and Description in a column -->
             <div class="flex flex-col flex-grow md:mr-4 mb-2 md:mb-0">
               <h4 class="item-name mb-0.5">{{ item.name[lang] }}</h4>
-              <p v-if="item.description && item.description[lang]" class="item-description">
-                {{ item.description[lang] }}
-              </p>
+              <div v-if="item.description && item.description[lang]">
```

Comment:
you are using a similar logic for both this and the CategoryList.vue component

Why not using a component for the items that you re-use when needed (DRY) ?